src/plugins/core/figures.ts

```diff
@@ -59,6 +69,31 @@ export class FigurePlugin extends CorePlugin<FigureState> implements FigureState
     }
   }
 
+  copy(state: ClipboardState, isCutOperation: boolean) {
+    if (state instanceof ClipboardFigureState) {
+      this.copiedFigure = state.copiedFigure;
+      this.copiedFigureContent = state.copiedFigureContent;
+      this.operation = isCutOperation ? "CUT" : "COPY";
+    }
+  }
```

Comment:
what I had in mind is that plugins would _return_ the clipped state and the clipboard itself would store it (without knowing exactly what's inside)
```suggestion
  copy(figureId, isCutOperation: boolean) {
    const copiedFigure = ...
    return {
       copiedFigure: ...
       copiedFigureContent: ...
    }
  }
```
Then, when calling the paste method, the clipboard would give the clipped content back
```ts
pasteFigure(sheetId: UID, position: { x: number; y: number }, clippedContent) {
   const copiedFigure = clippedContent.copiedFigure
   // do something
}
```